tools/example-showcase/window-settings-wasm.patch

```diff
@@ -1,11 +1,13 @@
 diff --git a/crates/bevy_window/src/window.rs b/crates/bevy_window/src/window.rs
-index 7b5c75d38..8e9404b93 100644
+index 87cdfb050..1d87a0bf5 100644
 --- a/crates/bevy_window/src/window.rs
 +++ b/crates/bevy_window/src/window.rs
-@@ -245,8 +245,8 @@ impl Default for Window {
+@@ -266,9 +266,9 @@ impl Default for Window {
              transparent: false,
              focused: true,
              window_level: Default::default(),
+-            fit_canvas_to_parent: false,
++            fit_canvas_to_parent: true,
              prevent_default_event_handling: true,
 -            canvas: None,
 +            canvas: Some("#bevy".to_string()),
```

Comment:
This will also not be needed if you use `WindowExtWebSys::canvas()`.

Reply:
Thanks for the insight!

You may be right about this specific example, but I think it's still ok to use that as a recommended way to use bevy, so the ability to target a specific canvas is in in bevy user space opposed to the lower level winit functions.

Reply:
For clarity, this "canvas" selector allows to declare the canvas at a specific place rather than relying on winit creating it